Cython/Compiler/ExprNodes.py

```diff
@@ -992,6 +991,13 @@ def coerce_to(self, dst_type, env):
               and src_type != dst_type
               and dst_type.assignable_from(src_type)):
             src = CoerceToComplexNode(src, dst_type, env)
+        elif (src.type.is_fastcall_type
+                and src_type != dst_type):
+            # These can always be coerced to a PyObject as a fallback option
+            # It may not be efficient but they're only intended to cover a
+            # limited number of cases efficiently
```

Comment:
```suggestion
            # These can always be coerced to a PyObject as a fallback option.
            # It may not be efficient but they're only intended to cover a
            # limited number of cases efficiently.
```